src/foam/comics/v2/DAOBrowserView.js

```diff
@@ -0,0 +1,184 @@
+/**
+ * @license
+ * Copyright 2019 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.comics.v2',
+  name: 'DAOBrowserView',
+  extends: 'foam.u2.View',
+  requires: [
+    'foam.u2.ActionView',
+    'foam.u2.dialog.Popup',
+    'foam.u2.layout.Cols',
+    'foam.u2.layout.Rows',
+    'foam.u2.search.Toolbar',
+    'foam.u2.view.ScrollTableView',
+    'foam.u2.view.TabChoiceView'
+  ],
+
+  documentation: `
+    A scrolling table view customized for the inline DAOController 
+    with canned queries and a searchbar
+  `,
+
+  css: `
+    ^export {
+      margin-left: 16px;
+    }
+
+    ^export img {
+      margin-right: 0;
+    }
+
+    ^top-bar {
+      border-bottom: solid 1px #e7eaec;
+      align-items: center;
+    }
+
+    ^query-bar {
+      padding: 24px 16px;
+      align-items: center;
+    }
+
+    ^toolbar {
+      flex-grow: 1;
+    }
+
+    ^browse-view-container {
+      margin: auto;
+      border-bottom: solid 1px #e7eaec;
+      margin: 20px 0px 72px 0px;
+    }
+
+    ^canned-queries {
+      padding: 0 16px;
+    }
+
+    /*
+      TODO: Don't manually target these classes like this.
+     */
+    ^ .foam-u2-view-ScrollTableView-table {
+      width: 100%;
+    }
+
+    ^ .foam-u2-view-ScrollTableView-scrollbarContainer {
+      height: 424px;
+    }
+
+    ^ .foam-u2-view-TableView th {
+      background: #ffffff
+    }
+
+    ^ .foam-u2-view-TableView td {
+      padding-left: 16px;
+    }
+  `,
+
+  imports: [
+    'stack?'
+  ],
+  exports: [
+    'dblclick'
+  ],
+  properties: [
+    {
+      class: 'foam.dao.DAOProperty',
+      name: 'data'
+    },
+    {
+      class: 'FObjectProperty',
+      of: 'foam.comics.v2.DAOControllerConfig',
+      name: 'config',
+      factory: function() {
+        return foam.comics.v2.DAOControllerConfig.create({ dao: this.data });
+      }
+    },
+    {
+      class: 'foam.mlang.predicate.PredicateProperty',
+      name: 'predicate',
+      expression: function(config$cannedQueries) {
+        return config$cannedQueries && config$cannedQueries.length 
+          ? config$cannedQueries[0].predicate
+          : foam.mlang.predicate.True.create();
+      }
+    },
+    {
+      class: 'foam.dao.DAOProperty',
+      name: 'predicatedDAO',
+      expression: function(config, predicate) {
+        return config.dao$proxy.where(predicate);
+      }
+    }
+  ],
+  actions: [
+    {
+      name: 'export',
+      label: '',
+      icon: 'images/export-arrow-icon.svg',
+      code: function() {
+        this.add(this.Popup.create().tag({
+          class: 'foam.u2.ExportModal',
+          exportData: this.predicatedDAO$proxy
+        }));
+      }
+    }
+  ],
+  methods: [
+    function dblclick(obj) {
```

Comment:
Let's name this something more meaningful to the intention rather than the specific user interaction.  Perhaps "onSelect"  The user may have selected it from the keyboard or by tapping on mobile or a tablet, onSelect explains in what abstract user action this is in response to, rather than what low level physical interaction happened.

Reply:
But let's also do this in another PR, merging as is :)